frontend/src/Components/CreateLendPage.tsx

```diff
@@ -0,0 +1,224 @@
+import { Box, Container, Paper, Typography, TextField, Button, Table, TableBody, TableCell, TableRow} from "@mui/material";
+import { useEffect, useState } from "react";
+import { Navigate, useNavigate } from "react-router-dom";
+import { ROUTER_PATHS } from "../Constants";
+import { ItemType } from "../Types/types";
+
+
+const defaultItem: ItemType = {
+	id: -1,
+	name: '',
+	lend_start: '',
+	lend_end: '',
+	img_uri: '',
+	is_lent_item: true
+}
+
+const defaultFormValues = {
+	name: '',
+	days: ''
+}
+
+const CreateLendPage = () => {
+
+	const navigation: any = useNavigate();
+
+	let [formValues, setFormValues] = useState(defaultFormValues)
+	let [item, setItem] = useState(defaultItem)
+
+	let [isCreated, setIsCreated] = useState(false)
+	let [title, setTitle] = useState("Create a Lend Item")
+	let [startIndex, setStartIndex] = useState(0)
+	let [endIndex, setEndIndex] = useState(0)
+
+	let [nameError, setNameError] = useState(false)
+	let [nameErrorMsg, setNameErrorMsg] = useState("")
+
+	let [daysError, setDaysError] = useState(false)
+	let [daysErrorMsg, setDaysErrorMsg] = useState("")
+
+
+	useEffect(()=>{
+		console.log("New item specs: {}",item)
+	},[item])
+
+
+	const onClickGoBack = () => {
+		navigation(ROUTER_PATHS.lend)
+	}
+
+
+	const handleInputChange = (event: any) => {
+    const {
+      id,
+      value
+    } = event.target;
+    setFormValues({
+      ...formValues,
+      [id]: value
+    });
+  };
+
+
+	const handleSubmit = () => {	
+		if(isValidSubmit()){
+			console.log("Form Values: {}",formValues);
+
+			const today = new Date()
+			const currentTime = 
+				today.getFullYear().toString()+"-"
+				+(today.getMonth()+1).toString()+"-"
+				+today.getDate().toString()+"T"
```

Comment:
I think this could be replaced by date.toISOString().
I used RFC339 for the dates in the db, if that helps with formatting.